### DISCUSSION TITLE
add back octavi os

### DISCUSSION THREAD

**Discussion by NRanjan-17** [ROM-builders/temporary#18073] [General]
@Apon77 sir please add octavi os back
https://github.com/ROM-builders/temporary/pull/18072
i confirmed its there main org now

**Comment by NRanjan-17** [comment]
@apon sir 
Did it need more time to merge octavi support in cirrus ?

**Comment by Apon77** [comment]
need time to be stable org

**Reply by NRanjan-17** [reply]
ok sir
